scripts/add-evita.py

```diff
@@ -26,7 +26,14 @@ def info(type, value, tb):
 
 
 def clamp_to_json_values(val):
-    return max(-1 * sys.float_info.max, min(val, sys.float_info.max))
+	# JSON doesn't have infinities, but we can use strings as long as we convert back.
+	if val == float('inf'):
+		return "Infinity"
+	elif val == float('-inf'):
+		return "-Infinity"
+	elif val == float('nan'):
+		return "NaN"
+	return val
```

Comment:
shouldn't we still clamp values to within [-sys.float_info.max, sys.float_info.max] AND use the strings for infinities/NaNs ?

Reply:
These are python floats, I don't think they can exceed those values without going to Infinity or -Infinity.

Well, technically you can exceed -1 * sys.float_info.max, but I don't see why that's a problem. If we're going to revert to the clamping code, we should probably use sys.float_info.min.

Reply:
well, let's not use sys.float_info.min since it is 2.2250738585072014e-308 ;)

But yeah I think you're right. The only purpose of the clamp was to avoid Infinities. this should be good as is